### DISCUSSION TITLE
What is the maximum speakers which can be identified ?

### DISCUSSION THREAD

**Discussion by amitli1** [pyannote/pyannote-audio#1297] [Development]
- What is the maximum number of speakers that the PyAnnote speaker diarization pipeline can find and separate ?
- Does that number depends on some assumptions ?

**Comment by Jasmine-fe** [comment]
max_speakers no upper limit (L63, diarization.py)

<img width="976" alt="Screenshot 2023-03-29 at 9 00 05 AM" src="https://user-images.githubusercontent.com/36653195/228399589-f205c457-4417-4586-b5bb-6125b38acc01.png">

**Reply by hbredin** [reply]
I confirm. There is no upper bound on the number of speakers (unless you provide one with the `max_speakers` option).
